bucky/src/IfElseStatement.java

```diff
@@ -0,0 +1,13 @@
+import java.util.Scanner;
+
+public class IfElseStatement {
+    public static void main(String[] args) {
+        Scanner bean = new Scanner(System.in);
```

Comment:
change variable name from `bean` to `sc`

Reply:
done